code-experiments/build/python/cython/interface.pyx

```diff
@@ -636,6 +637,7 @@ cdef class Problem:
     # cdef public np.ndarray upper_bounds
     cdef public np.ndarray _lower_bounds
     cdef public np.ndarray _upper_bounds
+    cdef public np.ndarray _largest_fvalues_of_interest
```

Comment:
I know it's just copy-paste, but the public qualifier doesn't look "right" for an attribute whose name starts with an underscore. Maybe it is historical left-over for the `_boundary...` attributes.

Reply:
It is, sorry.

Reply:
Seems to work if I remove the `public` keyword. Should I do this for `_lower_bounds` and `_upper_bounds` too?

Reply:
Some more information regarding public attributes (from http://cython.readthedocs.io/en/latest/src/reference/extension_types.html):

> To make attributes accessible to Python, they must be declared `public` or `readonly`

Since we don't need to access any of these three attributes starting by an underscore from Python, it should be okay to remove the `public` keyword for all three of them.